llvm/lib/Target/PowerPC/PPCISelLowering.cpp

```diff
@@ -3432,6 +3432,8 @@ SDValue PPCTargetLowering::LowerGlobalTLSAddressAIX(SDValue Op,
   if (Subtarget.hasAIXShLibTLSModelOpt())
     updateForAIXShLibTLSModelOpt(Model, DAG, getTargetMachine());
 
+  setUsesTOCBasePtr(DAG);
```

Comment:
Not setting the flag did not affect TLS functionality on AIX because "usesTOCBasePtr" was not called before this patch.
Therefore I think the change does not need to be put in a separate patch.